src/oop/static_modifier/medium/q3/MainTest.java

```diff
@@ -0,0 +1,230 @@
+package oop.static_modifier.medium.q3;
+
+import global.ObjectTest;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static global.tools.CustomAssertions._assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class MainTest {
+    private ObjectTest classInstance;
+    private ObjectTest testInstance;
+    private final String objectName = "Horse";
+    private final String testName = "TestAnimal";
+    private final String stringAttribute1 = "noise";
+    private final String stringAttribute2 = "name";
+    private final String methodName1 = "lengthenSound";
+    private final String methodName2 = "shortenSound";
+    private final String startNoise = "Neigh!";
+    private final String startVowel = "e";
+    private final int repetitionAmount = 4;
+    private final String testName1 = "Marly";
+    private final String testName2 = "Darling";
+
+    @BeforeEach
+    public void setup() {
+        String classPath = "oop.static_modifier.medium.q3." + objectName;
+        classInstance = new ObjectTest(classPath);
+        String testPath = "oop.static_modifier.medium.q3." + testName;
+        testInstance = new ObjectTest(testPath);
+    }
+
+    @Test
+    public void horseClassHasRequiredAttributes() {
```

Comment:
I think you will also need to test here that the static attribute is set to "Neigh" and you will probably have to order the tests as well.